src/seml/experiment/config.py

```diff
@@ -853,7 +875,20 @@ def determine_executable_and_working_dir(config_path, seml_dict):
             )
 
 
-def remove_prepended_dashes(param_dict):
+def remove_prepended_dashes(param_dict: dict[str, Any]) -> dict[str, Any]:
```

Comment:
Probably should be generic of T and map from `dict[str, T]` to `dict[str, T]`

Reply:
that's by definition a violation of the typing interface :D